exercise-8/README.md

```diff
@@ -10,12 +10,11 @@ With simple shell script:
 while sleep 0.5; do curl http://$INGRESS_IP/hello/world; done
 ```
 
-Or, with wrk2:
+Or, with fortio:
 
 ```sh
-docker pull saturnism/wrk2
-docker run -ti --rm saturnism/wrk2 \
-  -d 5s -R 5 http://$INGRESS_IP/hello/world
+docker run istio/fortio load -t 5m -qps 5 \
```

Comment:
changed from 5s in original to 5 minutes as it seems you'd want more than 5 seconds of load

it also seems simpler to just docker run than pull + run